theirc/rescue_id
# RI-13 30 minute session timeout

Applying this to all our own pages, but not
the admin (because apparently not all admin
pages load jQuery and it doesn't seem worth
fixing, given that we do not plan to expose
the admin to anyone but us).

I've added an empty page at / just so we can
test this. To test:

* Set short times in your local.py:

    # django-session-security
    SESSION_SECURITY_WARN_AFTER = 30  # seconds
    SESSION_SECURITY_EXPIRE_AFTER = 60

* Go to /admin and login
* Go to /
* Wait 30 seconds without touching anything and there should
  be a warning that the session will time out soon.
* Wait 30 more seconds and it should time out.
* Go to /admin and verify you need to login again. Login.
* Go to /
* Wait 30 seconds for the warning.
* Click in the page or move the mouse or something.
* Wait 30 more seconds and the login should not time out.

https://caktus.atlassian.net/browse/RI-13
